src/coreclr/src/vm/ecalllist.h

```diff
@@ -530,65 +530,65 @@ FCFuncStart(gDelegateFuncs)
 FCFuncEnd()
 
 FCFuncStart(gMathFuncs)
-    FCIntrinsicSig("Abs", &gsig_SM_Dbl_RetDbl, COMDouble::Abs, CORINFO_INTRINSIC_Abs)
-    FCIntrinsicSig("Abs", &gsig_SM_Flt_RetFlt, COMSingle::Abs, CORINFO_INTRINSIC_Abs)
-    FCIntrinsic("Acos", COMDouble::Acos, CORINFO_INTRINSIC_Acos)
-    FCIntrinsic("Acosh", COMDouble::Acosh, CORINFO_INTRINSIC_Acosh)
-    FCIntrinsic("Asin", COMDouble::Asin, CORINFO_INTRINSIC_Asin)
-    FCIntrinsic("Asinh", COMDouble::Asinh, CORINFO_INTRINSIC_Asinh)
-    FCIntrinsic("Atan", COMDouble::Atan, CORINFO_INTRINSIC_Atan)
-    FCIntrinsic("Atanh", COMDouble::Atanh, CORINFO_INTRINSIC_Atanh)
-    FCIntrinsic("Atan2", COMDouble::Atan2, CORINFO_INTRINSIC_Atan2)
-    FCIntrinsic("Cbrt", COMDouble::Cbrt, CORINFO_INTRINSIC_Cbrt)
-    FCIntrinsic("Ceiling", COMDouble::Ceil, CORINFO_INTRINSIC_Ceiling)
-    FCIntrinsic("Cos", COMDouble::Cos, CORINFO_INTRINSIC_Cos)
-    FCIntrinsic("Cosh", COMDouble::Cosh, CORINFO_INTRINSIC_Cosh)
-    FCIntrinsic("Exp", COMDouble::Exp, CORINFO_INTRINSIC_Exp)
-    FCIntrinsic("Floor", COMDouble::Floor, CORINFO_INTRINSIC_Floor)
-    FCFuncElement("FMod", COMDouble::FMod)
-    FCFuncElement("FusedMultiplyAdd", COMDouble::FusedMultiplyAdd)
-    FCFuncElement("ILogB", COMDouble::ILogB)
-    FCFuncElement("Log", COMDouble::Log)
-    FCFuncElement("Log2", COMDouble::Log2)
-    FCIntrinsic("Log10", COMDouble::Log10, CORINFO_INTRINSIC_Log10)
-    FCFuncElement("ModF", COMDouble::ModF)
-    FCIntrinsic("Pow", COMDouble::Pow, CORINFO_INTRINSIC_Pow)
-    FCFuncElement("ScaleB", COMDouble::ScaleB)
-    FCIntrinsic("Sin", COMDouble::Sin, CORINFO_INTRINSIC_Sin)
-    FCIntrinsic("Sinh", COMDouble::Sinh, CORINFO_INTRINSIC_Sinh)
-    FCIntrinsic("Sqrt", COMDouble::Sqrt, CORINFO_INTRINSIC_Sqrt)
-    FCIntrinsic("Tan", COMDouble::Tan, CORINFO_INTRINSIC_Tan)
-    FCIntrinsic("Tanh", COMDouble::Tanh, CORINFO_INTRINSIC_Tanh)
+    QCFuncElement("AbsInternal", COMDouble::AbsInternal)
```

Comment:
The FCall version used extended version of the macro that allowed specifying signature. This disambiguation won't happen anymore, and so the double entrypoint is going to be used for float as well. I think you may want to rename float Abs to AbsFInternal or something like that.

Reply:
I updated the float version to `AbsFInternal` in all places, but it has no effect on `IsJitIntrinsic` returning false for `Math.Abs(float)` in jitinterface.cpp

Reply:
My bad, I ran the test with old binaries. It has fixed the Abs(float) problem on x64. 👍